src/Microsoft.IIS.Administration/Program.cs

```diff
@@ -29,13 +30,18 @@ public class Program {
             using (var host = new WebHostBuilder()
                 .UseContentRoot(configHelper.RootPath)
                 .ConfigureLogging((hostingContext, logging) => {
+                    logging.AddConfiguration(hostingContext.Configuration.GetSection("Logging"));
+
                     //
                     // Console log is not available in running as a Service
                     if (!runAsAService) {
-                        logging.AddConfiguration(hostingContext.Configuration.GetSection("Logging"));
                         logging.AddConsole();
-                        logging.AddDebug();
-                    }
+                    }
+
+                    logging.AddDebug();
+                    logging.AddEventLog(new EventLogSettings() {

```

Comment:
Let's unify the event source with this
https://github.com/microsoft/IIS.Administration/blob/91ad2f8c4e206230f58fdf4e2a173bc2971e948b/src/Microsoft.IIS.Administration/Errors/ErrorHandler.cs#L43

Maybe poll out a constant

Reply:
Done

Reply:
We should make it a constant though

Reply:
Done.